tests/test_core.py

```diff
@@ -139,6 +139,12 @@ def can_do_standalone(self):
       '-fsanitize=address' not in self.emcc_args
 
 
+def also_with_wasmfs(func):
+  def decorated(self):
+    self.set_setting('WASMFS')
+  return decorated
```

Comment:
This doesn't have the behavior of "also" - it just tests with wasm FS. The `also_` family of decorators run the test once normally, and then once with an extra flag set, and I think that would make sense to do here too.